### PR TITLE
Add Java/Python solutions to CSES 1620 (Factory Machines)

### PR DESCRIPTION
**Pull request by jaredliw**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as
done._

- [x] I have tested my code
- [x] I have followed the code style guidelines mentioned
      [here](https://usaco.guide/general/adding-solution/#code-conventions)
- [x] I have properly formatted the files according to the steps mentioned
      [here](https://usaco.guide/general/adding-solution#steps)

### CODE REVIEW

File: solutions/cses-1620.mdx [hunk 1]
```diff
@@ -75,3 +79,84 @@ int main() {
...
+		int n = io.nextInt();  // number of machines
+		int goal = io.nextInt();
+
+		int[] machines = new int[n];
+		for (int i = 0; i < n; i++) {
+			machines[i] = io.nextInt();
+		}
+
+		long lo = 0;
+		long hi = 1000000000000000000L;  // 1e18
```
Comment (SansPapyrus683) [inline solutions/cses-1620.mdx:105]:
```suggestion
		long hi = (long) 1e18;
```

File: solutions/cses-1620.mdx [hunk 2]
```diff
@@ -75,3 +79,84 @@ int main() {
...
+</JavaSection>
+
+<PySection>
+
+```python
+_, goal = map(int, input().split())
+machines = list(map(int, input().split()))
+
+lo = 0
+hi = 1000000000000000000  # 1e18
```
Comment (SansPapyrus683) [inline solutions/cses-1620.mdx:143]:
```suggestion
hi = int(1e18)
```

File: solutions/cses-1620.mdx [hunk 3]
```diff
@@ -75,3 +79,84 @@ int main() {
...
+		long lo = 0;
+		long hi = 1000000000000000000L;  // 1e18
+		long ans = 0;
+		while (lo <= hi) {
+			long mid = (lo + hi) / 2;
+			
+			long n_produced = 0;
+			for (int machine : machines) {
+				n_produced += mid / machine;
+				// Prevent overflow
```
Comment (SansPapyrus683) [inline solutions/cses-1620.mdx:113]:
```suggestion
				// Break as soon as we hit our goal
```

**Review by SansPapyrus683 — CHANGES_REQUESTED** [review]

**Review by SansPapyrus683 — APPROVED** [review]

**Comment by jaredliw** [comment]
@SansPapyrus683 oops, i deleted the branch... I will open a new PR then...

**Comment by SansPapyrus683** [comment]
alr sure

**Pull request closed without merging**